docs/app/pages/blog/posts/2017-08-24-Protractor-Testing.md

```diff
@@ -0,0 +1,189 @@
+End-to-end (E2E) testing complements unit testing and does not replace it. It allows you to test the way users see your application.
```

Comment:
Possibly start with a brief explanation about what end to end tests are and then how they differ from unit tests?